lib/chai/utils/inspect.js

```diff
@@ -37,6 +37,7 @@ var isDOMElement = function (object) {
   } else {
     return object &&
       typeof object === 'object' &&
+      Object.hasOwnProperty.call(object, 'nodeType') &&
```

Comment:
This makes sense, but I will need to check if when using `typeof object.nodeType` it tries to access the property and therefore ends up throwing the proxy error we are trying to avoid.

I totally agree with you, we should support inheriting it from the prototype chain, but I'm not sure `typeof` would be the optimal solution.

Thanks for the review :smile: